LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.html

```diff
@@ -3,6 +3,10 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <style>
+textarea {
+  font-family: monospace;
```

Comment:
Remember to export this change to WPT.

This is needed because the test is reliant on font characteristics which aren't consistent across browsers.

Reply:
https://github.com/web-platform-tests/wpt/pull/44524